main.go

```diff
@@ -206,6 +206,21 @@ func NewApp() *cli.App {
 				return db.Wait()
 			}),
 		},
+		{
+			Name:  "wait-for-current",
+			Usage: "Wait for the database schema to become current",
+			Flags: []cli.Flag{
+				&cli.BoolFlag{
+					Name:    "quiet",
+					Aliases: []string{"v"},
```

Comment:
I would expect the flag `-v` to be verbose, and `-q` to be quiet.  Do you agree?